src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -143,35 +143,40 @@ const SubmissionDetails = _.flow(
     ])
   }
 
+  /**
+   * If the text has multiple lines, return just the first line of text with ellipses to indicate truncation.
+   */
+  const firstLine = text => {
+    const lines = text ? text.split('\n') : []
+    return lines.length > 1 ? `${lines[0]} ...` : lines.length > 0 ? lines[0] : undefined
+  }
```

Comment:
nit: We tend to prefer using lodash functions for consistency and for error handling. In this case `_.split` is better at handling bad values (e.g. number) on and will not throw an error. Additionally, these functions are auto curried.
```suggestion
  const firstLine = _.flow(
    _.split('\n'),
    lines => lines.length > 1 ? `${lines[0]} ...` : lines[0]
  )
```

Reply:
Thanks for the suggestion-- definitely simpler! I'm still unfamiliar with these functions, but will try to remember to look through them next time that I'm defining a utility method.